AdobeStockAsset/Model/Asset.php

```diff
@@ -53,15 +53,21 @@ public function setId($value): void
     /**
      * @inheritdoc
      */
-    public function getMediaTypeId(): int
+    public function getMediaTypeId(): ?int
     {
-        return (int) $this->getData(self::MEDIA_TYPE_ID);
+        $mediaTypeId = $this->getData(self::MEDIA_TYPE_ID);
+
+        if (!$mediaTypeId) {
```

Comment:
In this case, you'll get `null` even when you have some value set, for example, `0`
`$mediaTypeId === null` should be used instead or `$this->hasData(…)` method.

So it could look like
```php
return $this->hasData(self::MEDIA_TYPE_ID) ? (int)$this->getData(self::MEDIA_TYPE_ID) : null;
```

Reply:
Hi @hws47a 
Thanks for good suggestion!
The changes were applied for `setMediaTypeId ` and  `getPremiumLevelId` methods.